src/containers/User/UserGenerativeTokens.tsx

```diff
@@ -60,7 +60,7 @@ export function UserGenerativeTokens({
         onTrigger={handleFetchMore}
         canTrigger={!loading && !hasNothingToFetch}
       >
-        <CardsContainer>
+        <CardsContainer cardSize={270}>
```

Comment:
Since user creations don't have the cards explroerer we don't show card size select and provide the default card-size to the container